archunit/src/test/java/com/tngtech/archunit/core/domain/InstanceofCheckTest.java

```diff
@@ -0,0 +1,95 @@
+package com.tngtech.archunit.core.domain;
+
+import java.util.Iterator;
+
+import org.assertj.core.api.iterable.Extractor;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static com.tngtech.archunit.core.domain.TestUtils.importClasses;
+import static com.tngtech.archunit.core.domain.TestUtils.importMethod;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class InstanceofCheckTest {
```

Comment:
This test looks a little inconsistent to me. For example there is also no `ThrowsDeclarationTest`. ArchUnit does not have any convention, that "every class must have a unit test class" :wink: 
For example, you assert dependencies of `JavaClass`, so why are these tests here and not within `JavaClassTest`? There are already tests for other sorts of dependencies there, I would add it to that. But then I feel the first test asserting only one dependency provides no value against the detailed test in `DependencyTest` and thus there are no meaningful tests left :wink: